frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -132,9 +132,15 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
             },
         ],
         shouldShowEmptyState: [
-            (s) => [s.featureFlagsLoading, s.searchedFeatureFlags, s.searchTerm],
-            (featureFlagsLoading, searchedFeatureFlags, searchTerm): boolean => {
-                return searchedFeatureFlags && searchedFeatureFlags?.length == 0 && !featureFlagsLoading && !searchTerm
+            (s) => [s.featureFlagsLoading, s.searchedFeatureFlags, s.searchTerm, s.filters],
+            (featureFlagsLoading, searchedFeatureFlags, searchTerm, filters): boolean => {
+                return (
+                    searchedFeatureFlags &&
+                    searchedFeatureFlags?.length == 0 &&
+                    !featureFlagsLoading &&
+                    !searchTerm &&
+                    !filters.active
```

Comment:
Hey @nategrift , this same bug exists not just for the active filter, but any other kind of filter too. So, worth solving in this same PR.

(i.e. if you choose created by = person who didn't create any flags, same problem)

Reply:
Okay! Will do